man/git-authors.md

```diff
@@ -24,11 +23,11 @@ git-authors(1) -- Generate authors report
 
 ## EXAMPLES
 
-  * Updating AUTHORS file:
+  Updating AUTHORS file:
 
     $ git authors
 
-  * Listing authors:
+  Listing authors:
```

Comment:
I think about it again. Maybe we can remove the '```' from the markdown, as the output should be in the same context as the command line `$ git xxx`. What do you think?

Reply:
Make sense when try to invoke command then output the response in the ternimal, but how about previewing the man page. Take a wait and I do a little test.

Reply:
Great, it is ok and there is a precedent `git-alias` already...

Reply:
Done